data/campaigns/Secrets_of_the_Ancients/gui/zombie_recruit_dialog.cfg

```diff
@@ -168,7 +168,7 @@
                 [label]
                     definition ="default_small"
                     use_markup = "true"
-                    label = "<b>" .. _ "Traits" .. "</b>"
+                    label = "<b>" _ "Traits" "</b>"
```

Comment:
These dots should be replaced with `+`. It was probably an error from converting the dialog to Lua.